testing/architecture-test/src/changes/accepted-changes/processForkOptions-accepted-public-api-changes.json

```diff
@@ -0,0 +1,12 @@
+{
+    "acceptedApiChanges": [
+        {
+            "type": "org.gradle.kotlin.dsl.GradleApiKotlinDslExtensions_4i8m7ubpazumv0o7fg1f6fdogKt",
```

Comment:
:question: What's that mangled name doing in here? 🤮

Reply:
It's from generated accessors